vault/identity_store_oidc_test.go

```diff
@@ -1097,59 +1096,52 @@ func TestOIDC_PeriodicFunc(t *testing.T) {
 			setSigningKey:     false,
 			setNextSigningKey: false,
 			testCases: []testCase{
-				{1, 0, 0},
-				{2, 2, 2},
+				{0, 0},
+				{2, 2},
 			},
 		},
 	}
 
 	for _, testSet := range testSets {
 		testSet := testSet
 		t.Run(testSet.namedKey.name, func(t *testing.T) {
-			// We've disabled parallelism because this test is sensitive to timing and not running it in parallel
-			// makes it less flaky.
-			// t.Parallel()
+			t.Parallel()
```

Comment:
I'm not exactly sure how this could have timing conflicts (see prior comment) given that each test set has its own unsealed core and storage context. My only guess is that running in parallel could introduce resource contention on the test runner thereby affecting timing.